cg/meta/archive/ddn/constants.py

```diff
@@ -0,0 +1,51 @@
+from enum import StrEnum
+
+OSTYPE: str = "Unix/MacOS"
+ROOT_TO_TRIM: str = "/home"
+
+DESTINATION_ATTRIBUTE: str = "destination"
+SOURCE_ATTRIBUTE: str = "source"
+
+
+class DataflowEndpoints(StrEnum):
+    """Enum containing all DDN dataflow endpoints used."""
+
+    ARCHIVE_FILES = "files/archive"
+    DELETE_FILE = "files/delete"
+    GET_AUTH_TOKEN = "auth/token"
+    REFRESH_AUTH_TOKEN = "auth/token/refresh"
+    RETRIEVE_FILES = "files/retrieve"
+    GET_JOB_STATUS = "activity/jobs/"
+
+
+class JobStatus(StrEnum):
+    """Enum for the different job statuses which can be returned via Miria."""
+
+    CANCELED = "Canceled"
+    COMPLETED = "Completed"
+    DENIED = "Denied"
+    CREATION_IN_PROGRESS = "Creation in progress"
+    IN_QUEUE = "In Queue"
+    INVALID_LICENSE = "Invalid license"
+    ON_VALIDATION = "On validation"
+    REFUSED = "Refused"
+    RUNNING = "Running"
+    SUSPENDED = "Suspended"
+    TERMINATED_ON_ERROR = "Terminated on error"
+    TERMINATED_ON_WARNING = "Terminated on warning"
+
+
+FAILED_JOB_STATUSES: list[str] = [
+    JobStatus.CANCELED,
+    JobStatus.DENIED,
+    JobStatus.INVALID_LICENSE,
+    JobStatus.REFUSED,
+    JobStatus.TERMINATED_ON_ERROR,
+    JobStatus.TERMINATED_ON_WARNING,
+]
+ONGOING_JOB_STATUSES: list[str] = [
```

Comment:
Not in this PR, but it would be nice to make these classmethods of JobStatus instead of detached constants.

Reply:
That is a nice suggestion, I will do that in a future PR.